docs/decisions/0037-audio-naming.md

```diff
@@ -0,0 +1,103 @@
+
+### [Implementation]
+
+As for implementations, there are two options as well - keep it as it is or rename classes based on how the capability is called by AI provider and most probably renaming is the best choice here, because from the user point of view, it will be easier to understand which concrete OpenAI capability is used (e.g. `transcription` or `translation`), so it will be easier to find related documentation about it and so on.
+
+Proposed renaming:
+
+- AzureOpenAIAudioToTextService -> AzureOpenAIAudioTranscriptionService
+- OpenAIAudioToTextService -> OpenAIAudioTranscriptionService
+- AzureOpenAITextToAudioService -> AzureOpenAITextToSpeechService
+- OpenAITextToAudioService -> OpenAITextToSpeechService
+
+## Naming comparison
+
+| AI Provider  | Audio conversion    | Proposed Interface         | Proposed Implementation             |
+| ------------ | ------------------- | -------------------------- | ----------------------------------- |
+| Microsoft    | Text-to-speech      | ITextToSpeechService       | MicrosoftTextToSpeechService        |
+| Microsoft    | Speech-to-text      | IAudioTranscriptionService | MicrosoftSpeechToTextService        |
+| Microsoft    | Speech translation  | IAudioTranslationService   | MicrosoftSpeechTranslationService   |
+| OpenAI       | Text-to-speech      | ITextToSpeechService       | OpenAITextToSpeechService           |
+| OpenAI       | Audio transcription | IAudioTranscriptionService | OpenAIAudioTranscriptionService     |
+| OpenAI       | Audio translation   | IAudioTranslationService   | OpenAIAudioTranslationService       |
+| Google       | Speech-to-text      | IAudioTranscriptionService | GoogleSpeechToTextService           |
+| Google       | Text-to-speech      | ITextToSpeechService       | GoogleTextToSpeechService           |
+| Amazon       | Text-to-speech      | ITextToSpeechService       | AmazonTextToSpeechService           |
+| Amazon       | Transcription       | IAudioTranscriptionService | AmazonAudioTranscriptionService     |
+| Meta         | Text-to-sound       | TBD                        | TBD                                 |
+| Meta         | Text-to-music       | ITextToMusicService        | MetaTextToMusicService              |
+| Hugging Face | Text-to-speech      | ITextToSpeechService       | HuggingFaceTextToSpeechService      |
+| Hugging Face | Text-to-audio       | TBD                        | TBD                                 |
+| Hugging Face | Audio-to-audio      | TBD                        | TBD                                 |
+| Hugging Face | Speech recognition  | IAudioTranscriptionService | HuggingFaceSpeechRecognitionService |
+| AssemblyAI   | Transcription       | IAudioTranscriptionService | AssemblyAIAudioTranscriptionService |
```

Comment:
Could you arrange this table by "Audio conversion", as I think it's useful to see all of the same service implementations together

Reply:
@markwallace-microsoft Done. You can also see that by doing that the values in `Proposed interface` column became grouped as well, since we use the same interface for same/similar type of audio conversion.